### ISSUE TITLE
Install failure on Ubuntu 20.04.3

### ISSUE THREAD

**Issue by JonathanHUnity** [GitCredentialManager/git-credential-manager#566]
I got the following error when installing `gcmcore-linux_amd64.2.0.605.12951.deb` on Ubuntu 20.04.3:
```
$ sudo dpkg -i ~/Downloads/gcmcore-linux_amd64.2.0.605.12951.deb 
Selecting previously unselected package gcmcore.
(Reading database ... 246246 files and directories currently installed.)
Preparing to unpack .../gcmcore-linux_amd64.2.0.605.12951.deb ...
Unpacking gcmcore (2.0.605.12951) ...
Setting up gcmcore (2.0.605.12951) ...
```

After this, `git-credential-manager-core configure` failed with `git-credential-manager-core: command not found`

It appears this installer adds files to /urs/local/share/gcm-core/Release, but the link in /usr/local/bin expects files in /usr/local/share/gcm-core
![image](https://user-images.githubusercontent.com/23556416/145605002-24b12431-4b53-4bbb-8e4e-0f3d9e916a9f.png)

**Comment by ldennington** [comment]
Hi @JonathanHUnity! This was actually fixed with #539 but has not yet been released. The current workaround is to run:

`sudo mv /usr/local/share/gcm-core/Release/* /usr/local/share/gcm-core/ && sudo rm -rf /usr/local/share/gcm-core/Release/`

**Comment by JonathanHUnity** [comment]
Ok! I worked around it using the tgz.
